src/components/results/index.tsx

```diff
@@ -58,6 +58,16 @@ export default class Results extends Component<Props, State> {
     this.props.onCopyToOtherClick();
   }
 
+  @bind
+  onDownload() {
+    ga('send', 'event', 'compression', 'download', {
```

Comment:
You don't have an event label here. It might be interesting to set the file type as the event label. Then you could report on which file types got the most savings.